.github/workflows/dataflow_engine_chaos.yaml

```diff
@@ -3,6 +3,8 @@ name: Dataflow Engine Chaos
 on:
   schedule:
     - cron: '0 17-23 * * *' # run at minute 0 every hour from 01:00 ~ 07:00 UTC+8
+  pull_request:
```

Comment:
it's to test the effect of this PR, or we are really going to let every PR trigger chaos test?

Reply:
It is for test only.